assets/js/screens/Work/Work.jsx

```diff
@@ -59,31 +59,39 @@ const ScreensWork = () => {
   if (error) return <Error error={error} />;
 
   const {
-    work: { accessionNumber, published, descriptiveMetadata, project, sheet }
+    work: { accessionNumber, published, descriptiveMetadata, project, sheet },
   } = data;
 
   const breadCrumbs = [
+    {
+      label: `Projects`,
+      route: `/project/list`,
+    },
     {
       label: `${project.name}`,
-      route: `/project/${project.id}`
+      route: `/project/${project.id}`,
+    },
+    {
+      label: "Ingest Sheets",
+      route: `/project/${id}`,
```

Comment:
This is passing work ID and not project ID as expected.

<img width="1204" alt="Screen Shot 2020-04-29 at 3 20 59 PM" src="https://user-images.githubusercontent.com/14085957/80643094-252ba480-8a2d-11ea-91e8-5d4f35acc581.png">

Reply:
So, the logic is, though it might not be good logic, is that "Ingest Sheets" refers to a list of Ingest Sheets, which technically live on an individual Project screen.   Make sense?

Agreed we need a good/better breadcrumb strategy for nested content to make it obvious to users where they're at in the app, and ideally reinforce the association between Projects, Ingest Sheets, Works, and Filesets.

I'll kick off a ticket and get @davidschober 's thoughts.

Reply:
Ok, I think we can remove it from this coz it is throwing info not found error on the page. If you go into a work and click on "Ingest sheets", it should reproduce that error.